trftools/dictionaries/_subtlex.py

```diff
@@ -31,20 +31,28 @@ def read_subtlex(lower=False):
     Word, FREQcount, CDcount, FREQlow, Cdlow, SUBTLWF, Lg10WF, SUBTLCD, Lg10CD
 
     """
-    path = Path(__file__).parent / 'data' / 'SUBTLEXus74286wordstextversion.txt'
-    if not path.exists():
-        download('https://www.ugent.be/pp/experimentele-psychologie/en/research/documents/subtlexus/subtlexus2.zip/at_download/file', path, unzip=True)
+    if language == 'ENG':
+        path = Path(__file__).parent / 'data' / 'SUBTLEXus74286wordstextversion.txt'
+        if not path.exists():
+            download('https://www.ugent.be/pp/experimentele-psychologie/en/research/documents/subtlexus/subtlexus2.zip/at_download/file', path, unzip=True)
+    elif language == 'NL':
+        path = Path(__file__).parent / 'data' / 'SUBTLEX-NL.cd-above2.txt'
+
+
     out = {}
     str_trans = str.lower if lower else str.upper
     with path.open() as fid:
         columns = fid.readline().split()
         i_key = columns.index('Word')
         columns.pop(i_key)
-        for line in fid:
+        for i, line in enumerate(fid):
```

Comment:
Since `i` is not actually needed it should not be created, as it will make the function slower.